### DISCUSSION TITLE
Coupled ensemble HR4 test case on Hercules?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/global-workflow#2940] [Q&A] [answered]
Hello,

I would like to try setting up a coupled, ensemble test case using HR4 on Hercules, to make sure I have something working before attempting to port to my local cluster and ultimately Frontera. Ideally this would be the S2S configuration at C192mx025, but really any S2S configuration would do. Is HR4 set up for a case like this, and if so what IC files/configurations are available?

Thanks!

**Comment by benjamin-cash** [comment]
@JessicaMeixner-NOAA - I seem to recall seeing a recent update from you about where to find ICs for HR4, would you be able to point me in the right direction? At the moment I would settle for any set of ICs that function with HR4 so that I can test my setup.

**Comment by jiandewang** [comment]
@benjamin-cash here I copy and paste the IC location for HR4 (C1152 S2SW run)
IC Locations

Ocean/Ice/Wave: 
WCOSS2: /lfs/h2/emc/couple/noscrub/Jiande.Wang/IC/HR3marine
HERA: /scratch1/NCEPDEV/climate/role.ufscpara/IC/HR3marine
Orion: /work/noaa/marine/jmeixner/IC/HR3marine
HPSS: /NCEPDEV/emc-climate/5year/role.ufscpara/IC/HR3/HR3marine

Atm: C1152
WCOSS2: /lfs/h2/emc/couple/noscrub/Jiande.Wang/IC/HR4C1152
HERA: /scratch1/NCEPDEV/climate/role.ufscpara/IC/HR4C1152
Orion: 
HPSS:  /NCEPDEV/emc-climate/5year/role.ufscpara/IC/HR4/HR4C1152

**Reply by benjamin-cash** [reply]
Thanks @jiandewang ! Of those systems I only have access to Orion, would it be possible to copy the atm ICs to Orion/Hercules, or the AWS bucket?

**Reply by jiandewang** [reply]
you may contact @WalterKolczynski-NOAA as usually he copy data to orion/hercules

**Reply by benjamin-cash** [reply]
Thanks again @jiandewang - I'll look for @WalterKolczynski-NOAA 's response.

**Reply by KateFriedman-NOAA** [reply]
For the HR ICs we usually just symlink to them being staged under either @jiandewang's or @JessicaMeixner-NOAA's or role.ufscpara space. @jiandewang can the HR4 ICs be copied to one of those accounts on Orion/Hercules? Then we can symlink to them just like we do on Hera:

```
[role.glopara@hfe02 prototype_ICs]$ pwd
/scratch1/NCEPDEV/global/glopara/data/ICSDIR/prototype_ICs
[role.glopara@hfe02 prototype_ICs]$ ll | grep HR4
lrwxrwxrwx 1 role.glopara global    51 Aug 29 12:36 HR4C1152 -> /scratch1/NCEPDEV/climate/role.ufscpara/IC/HR4C1152
```

**Reply by WalterKolczynski-NOAA** [reply]
@JessicaMeixner-NOAA copied the data for HR3 (which we already have links to), so it would just be the updated ATM ICs for HR4. We then may need to create a symlink tree to put it in the right structure.

**Comment by jiandewang** [comment]
@WalterKolczynski-NOAA do we have HR3 C1152 atmos IC on orion ? if yes can you point me to the path ?

**Comment by jiandewang** [comment]
I saw orion /work/noaa/marine/jmeixner/IC/HR3C1152
this means @JessicaMeixner-NOAA coiped all HR3 C1152 atmos IC already, so I am going to copy HR4 atmso IC to orion

**Reply by WalterKolczynski-NOAA** [reply]
Yup, those are the ones.

**Reply by benjamin-cash** [reply]
Thanks again @jiandewang :) Please let me know when they are copied over and where to find them.

**Comment by jiandewang** [comment, marked as answer]
C1152 HR4 atmos IC on Orion: /work/noaa/marine/Jiande.Wang/IC/HR4C1152

**Reply by benjamin-cash** [reply]
Thank you!
